interfaces/policy/helpers.go

```diff
@@ -81,15 +81,26 @@ func checkOnClassic(c *asserts.OnClassicConstraint) error {
 }
 
 func checkPlugConnectionConstraints1(connc *ConnectCandidate, cstrs *asserts.PlugConnectionConstraints) error {
```

Comment:
I don't think we really want to change this code like this, way really need to check the final set of attributes, merged as it makes sense, the rules are about attributes as whole, so checking subsets separetely doesn't have the same meaning